### PR TITLE
Survey-responsive

### PR DESCRIPTION
**Pull request by fang-helen**
Made the bubbles on the survey page "select" and "unselect" when clicked, where only 1 bubble can be selected per survey question.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -303,3 +308,22 @@ async function getEvents(url, index, option) {
...
+ * Toggles the display of the survey page to indicate which option is selected,
+ * while saving the survey responses.
+ *
+ * @param {number} question The question number to apply the toggle.
+ * @param {number} index Which bubble on the question to apply the toggle to.
+ */
+var surveyResponses = [-1, -1, -1, -1, -1];
+function toggleSurveyDisplay(question, index) {
+	const circle = document.getElementsByClassName('survey-select')[question*5 + index];
+	if(surveyResponses[question] >= 0) {
```
Comment (0tpyrk) [inline src/main/webapp/script.js:322]: Add space between if and (

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -303,3 +308,22 @@ async function getEvents(url, index, option) {
...
+		var oldIndex = surveyResponses[question];
+		const oldCircle = document.getElementsByClassName("survey-select")[question*5 + oldIndex];
+		oldCircle.style.backgroundColor = "transparent";
+	}
+	circle.style.backgroundColor = "white";
```
Comment (0tpyrk) [inline src/main/webapp/script.js:327]: In JS I think we're supposed to use single quotes instead of double

File: src/main/webapp/script.js (hunk 1 above)
Comment (fang-helen) [reply src/main/webapp/script.js]: I keep forgetting, thanks!

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -392,6 +396,7 @@ input[type=text], textarea {
 }
 
 .event-right-details {
+	/* right side with date, time, location */
```
Comment (nmsulliv) [inline src/main/webapp/style.css:399]: Maybe get rid of the extra tab. Everything looks good!

File: src/main/webapp/script.js (hunk 2 above)
Comment (tc466) [reply src/main/webapp/script.js]: Maybe useful to use a linter to automatically check the code against the style guide:

https://github.com/eslint/eslint
https://github.com/google/eslint-config-google

**Pull request merged**